### DISCUSSION TITLE
Log4j Security Vulnerability

### DISCUSSION THREAD

**Discussion by zurawv** [open-telemetry/opentelemetry-collector#4553] [Q&A] [answered]
I would like to ask if the opentelemetry-collector is also affected of the log4j security issue?
I couldn't find any information about this in the repos, so i hope it's okay to ask the question here.

thank you in advance :)

**Comment by Aneurysm9** [comment, marked as answer]
The OpenTelemetry Collector is implemented in Go and does not use log4j.
